### DISCUSSION TITLE
2024-05-17 - 🦙 Supporting the announcement of the Tina.io Acquisition - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#8557] [Chewing the Fat]
What are your thoughts on SSW acquiring Tina.io?

- https://www.linkedin.com/posts/tinacms_markdown-github-cms-activity-7198196858869997570-QK2-
- https://www.linkedin.com/posts/ssw_markdown-github-cms-activity-7196762427642789889-dVVz
- https://www.linkedin.com/posts/adamcogan_blogged-ssw-purchases-tinacms-the-1-cms-activity-7196403849752911872-qNDY

**Comment by tkapa** [comment]
It's an exciting new time for SSW. I'm hoping to jump in and learn a bunch of new things with the Tina Team in the future.

**Comment by PothieuG** [comment]
I'm looking forward how will evolve with SSW at the helm!
Super excited!

**Comment by matt-goldman-ssw** [comment]
Wish I could've seen the buzz at ./NEW!

**Comment by drwharris** [comment]
Looks like you all had fun. Looking forward to seeing the new features coming.

**Comment by bettybondoc** [comment]
Working as a designer on a playful brand like Tina's is quite exciting. It's a stark contrast to the strong and masculine brand of SSW. 💪🏼🔴⚫️ I like to think of Tina as the cute, rebellious, and charming daughter of SSW. 🦙🧡💙



![Asset 8 8](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33026270/1528ba1e-eafb-4b5a-b3aa-9a40c98d08ce)
**Figure: Guilty of using a lot of blue 💙**

**Reply by leahy268** [reply]
I want a one of those mugs :)

**Reply by bettybondoc** [reply]
@camillars 👆🏼

**Reply by tomek-i** [reply]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/6b901975-0718-49e5-a052-96743a92b9a6)

**Reply by JeanThirion** [reply]
Can I also get a mug shipped to Strasbourg ?? 🙏🙏

**Reply by JackDevAU** [reply]
+1 on the mug!! 

Loved all your Tina designs!! Amazing work!!

**Reply by PennyWalker** [reply]
I want that mug so bad..... :D

**Comment by jakebayliss** [comment]
Exciting times!

**Comment by ozdemir-mehmet-cer** [comment]
Ditto

**Comment by leahy268** [comment]
Exciting times but maybe a bit scary :) - Lot's of work to do..

**Comment by alexleessw** [comment]
🎉🎉🍾🍾

**Comment by Tanya-Leahy** [comment]
Very exciting time for SSW!

**Comment by SamBWagner** [comment]
It's pretty exciting!

**Comment by Ben0189** [comment]
exciting news🦙🦙🦙

**Comment by joshbermanssw** [comment]
🦙 !! Very exciting new horizons

**Comment by kikibianc** [comment]
As everyone said, very exciting indeed!
Time to experience the love and hate of the crowd with a product like Tina, where we're very exposed to the elements - smooth seas never made a skilled sailor 😎

**Comment by uly1** [comment]
My usual misgivings about splitting our focus are still there... but it's a great opportunity and we might as well take it by the horns... do llamas have horns?

**Comment by steven0x51** [comment]
it's really an exciting time!🎉🍾

**Comment by ozair-ashfaque** [comment]
Really excited about Tina 🤞💖

**Comment by camillars** [comment]
Exciting times for SSW, the brand is fun! Can't wait to see where it will take us - Lots of hard work ahead! 💪

**Comment by YalalCakirSSW** [comment]
So excited for SSW 🦙🧡

**Comment by MarcusIrmscherSSW** [comment]
There is a lot of potential for Tina, I hope we can capitalize on it. 💲

**Comment by duanxinhuan** [comment]
Good to know we have a conference for Tina.
Keep moving

**Comment by awaltos** [comment]
Great move! I think this acquisition is beneficial for both parties. For SSW to inject some excitement and change and building on an already solid foundation of contributing to OSS. And for Tina to gain resources and experience from a 30+ year strong company to build on the great work they have already achieved. Looking forward to being a part of it all.

**Comment by AntPolkanov** [comment]
Tina has great potential, and I am looking forward to seeing what it will become in the future.

**Comment by TobyChurches** [comment]
Its great news! Looking forward to seeing what it will become 🙂

**Comment by AttackOnMorty** [comment]
Awesome 🦙 😘

**Comment by jimmy1992abc** [comment]
![20240515_184143](https://github.com/SSWConsulting/SSW.Rules.Content/assets/112846253/00977c65-3d73-43a4-a3e7-41c8ca1fac0b)

**Comment by chrisschultzssw** [comment]
Exciting times for SSW, Tina, and llamas! 🦙

**Comment by Marxoz** [comment]
This is going to be fun!

**Comment by ricksu978** [comment]
⠀⠀⠀⠀⡾⣦⡀⠀⠀⡀⠀⣰⢷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⣠⠗⠛⠽⠛⠋⠉⢳⡃⢨⢧⣄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⣰⠋⠁⠀⠀⠀⠀⠀⠀⠙⠛⢾⡈⡏⢧⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⣼⠉⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⢧⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠸⢦⡀⠀⠀⠀⠀⢀⠀⠀⠀⠀⠀⠀⠀⠀⢸⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⢈⠟⠓⠶⠞⠒⢻⣿⡏⢳⡀⠀⠀⠀⠀⢸⡆⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⡴⢉⠀⠀⠀⠀⠀⠈⠛⢁⣸⠇⠀⠀⠀⠀⢺⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⢧⣸⡁⠀⠀⣀⠀⠀⣠⠾⠀⠀⠀⠀⠀⠀⣹⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠉⠓⢲⠾⣍⣀⣀⡿⠃⠀⠀⠀⠀⠀⠀⢸⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣇⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⣏⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⢀⡗⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡼⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⢸⡄⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢟⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⣸⠂⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠺⠦⠤⠤⣤⣄⣀⣀⡀⠀⠀⠀⠀⠀
⠀⠀⠀⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠉⠉⠳⣦⣄⠀⠀
⠀⠀⢀⡷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠻⣆⠀
⠀⠀⣼⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠘⣆
⠀⠀⣏⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣿
⠀⠀⢹⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣼
⠀⠀⠀⣏⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡞
⠀⠀⠀⠈⢷⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⡇
⠀⠀⠀⠀⠈⢻⣦⣀⠀⣏⠀⠀⠀⠀⠀⠀⢸⡆⠀⠀⢠⡄⠀⠀⠀⠀⠀⢀⡿⠀
⠀⠀⠀⠀⠀⠀⠻⡉⠙⢻⡆⠀⠀⠀⠀⠀⡾⠚⠓⣖⠛⣧⡀⠀⠀⠀⢀⡾⠁⠀
⠀⠀⠀⠀⠀⠀⠀⠙⡇⢀⡿⣦⡀⠀⢀⡴⠃⠀⠀⠈⣷⢈⠷⡆⠀⣴⠛⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠛⠚⠀⢸⡇⣰⠏⠁⠀⠀⠀⠀⢉⠁⢸⠷⠼⠃⠀⠀⠀⠀

**Comment by jeoffreyfischer** [comment]
Let's make CMS great again! Matt W's demo was awesome at the SSW Newcastle party!! Excited to see how TinaCMS will be in a few months :)

**Comment by tomek-i** [comment]
A while back I wanted to update my gh-pages as I never really did anything with it at all.
I developed a small SPA in react that reads *.md files out. parses front-matter and generates a timeline 
from those documents - i haven't switched the branches yet because I haven't finalized the design yet.

After seeing the demo of TinaCMS at the Newcastle office I wanna change it over and use that instead and see how it goes :-)
                                                                                                                                                  
                                        ░░░░░░░░░░                                              ░░░░░░░░░░                                        
                                        ░░░░  ░░░░                                              ░░░░  ░░░░                                        
                                      ░░░░░░    ░░                                              ░░    ░░░░░░                                      
                                      ░░▒▒▒▒    ░░░░                                          ░░░░    ▒▒▒▒░░                                      
                                      ░░▒▒▒▒▒▒  ░░░░                                          ░░░░  ▒▒▒▒▒▒░░                                      
                                      ░░▒▒▒▒▒▒░░  ░░░░                                      ░░░░░░░░▒▒▒▒▒▒░░                                      
                                      ░░▒▒▒▒▒▒▒▒░░░░░░                                      ░░░░░░▒▒▒▒▒▒▒▒░░                                      
                                      ░░░░▒▒▒▒░░░░░░▒▒                                      ▒▒░░░░░░▒▒▒▒░░░░                                      
                                      ░░░░▒▒▒▒▒▒░░░░▒▒░░                                  ░░░░░░░░▒▒▒▒▒▒░░░░                                      
                                        ░░▒▒▒▒▒▒▒▒░░░░░░                                  ░░░░░░▒▒▒▒▒▒▒▒░░                                        
                                        ░░▒▒▒▒░░░░░░░░░░                                  ░░░░░░░░░░▒▒▒▒░░                                        
                                          ░░▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒░░░░░░▒▒▒▒▒▒░░                                          
                                          ░░░░▒▒▒▒░░░░░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒░░░░░░▒▒▒▒░░░░                                          
                                            ▒▒▒▒░░░░░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒░░░░░░▒▒▒▒                                            
                                            ░░░░░░░░▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒░░░░░░░░                                            
                                              ▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒                                              
                                              ▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒░░░░░░░░░░░░░░░░░░▒▒▒▒                                              
                                              ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░                                              
                                              ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░                                              
                                              ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░                                              
                                              ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░                                              
                                              ░░▒▒▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒▒▒░░                                              
                                          ░░▒▒▒▒▒▒▓▓██▓▓▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▓▓▓▓▓▓▒▒▒▒▒▒░░                                          
                                            ░░▒▒▒▒██░░░░▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒██▒▒  ▒▒▒▒░░                                            
                                                ▓▓▓▓████▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒██████▓▓                                                
                                                ░░██▓▓██▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒██▓▓██░░                                                
                                                ░░▒▒▒▒▒▒▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▒▒▒▒▒▒░░                                                
                                              ░░░░░░▒▒▒▒░░░░░░░░░░░░    ░░      ░░░░░░░░░░▒▒▒▒░░░░░░                                              
                                              ░░░░░░▒▒░░░░░░░░░░                  ░░░░░░░░░░▒▒░░░░░░                                              
                                              ░░░░░░░░░░░░░░░░░░░░▒▒▒▒      ▒▒▒▒░░░░░░░░░░░░░░░░░░░░                                              
                                              ░░░░░░░░░░░░░░░░░░██████▒▒▒▒▒▒██████░░░░░░░░░░░░░░░░░░                                              
                                              ░░░░▒▒░░░░░░░░░░░░▒▒████▒▒▒▒▒▒████▒▒░░░░░░░░░░░░░░░░░░                                              
                                              ░░░░░░░░░░░░░░░░    ░░██▒▒▒▒▒▒██▒▒  ░░░░░░░░░░░░░░░░░░                                              
                                        ░░      ░░░░░░░░░░░░░░░░    ░░▒▒▒▒▒▒░░      ░░░░▒▒░░░░░░░░          ░░                                    
                                    ░░░░  ░░░░░░░░░░░░░░░░▒▒░░░░░░  ░░░░▒▒  ░░░░░░░░  ░░▒▒░░░░░░░░░░  ░░░░  ░░  ░░                                
                                  ░░    ░░    ▒▒▒▒░░░░▒▒░░▒▒  ░░  ░░    ▓▓░░    ░░    ░░░░░░▒▒░░░░░░░░      ░░░░                                  
                                    ░░░░░░    ░░▒▒░░░░▒▒░░▒▒░░▒▒  ░░░░▒▒▒▒▒▒      ░░░░░░░░░░░░░░░░░░░░      ░░░░                                  
                                      ░░░░  ░░▓▓░░░░░░░░░░▒▒░░░░▓▓        ░░    ▓▓  ▒▒░░▒▒░░░░░░░░▓▓▒▒      ░░                                    
                                    ░░    ░░▓▓▒▒░░░░░░░░░░░░░░  ▒▒██▒▒▒▒▒▒▒▒▒▒▓▓▒▒░░░░░░░░░░░░░░░░▓▓▓▓░░                                          
                                          ▓▓▓▓▓▓░░░░░░░░░░░░░░░░    ██▓▓██▓▓▓▓    ░░░░░░░░░░░░░░░░▓▓▓▓▒▒                                          
                                          ▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░    ░░░░░░░░  ░░░░░░░░░░░░░░░░░░▓▓▓▓▒▒                                          
                                        ▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░          ░░░░░░░░░░░░░░░░░░▒▒▓▓▓▓▓▓                                          
                                        ▓▓▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░░░░░░░░░░░▒▒░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓                                          
                                        ▒▒▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▓▓▓▓▓▓                                          
                                        ░░▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▒▒                                          
                                        ▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓░░                                          
                                        ▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓                                            
                                        ▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▓▓▓▓░░                                              
                                      ░░▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▒▒▓▓▓▓▒▒                                              
                                      ░░▓▓▓▓▒▒▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓▒▒                                              
                                        ▓▓▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓▓▓▒▒                                              
                                        ▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓▓▓▓▓                                                
                                          ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░░░░░░░░░░░░░░░░░░░░░▓▓▓▓▓▓▓▓▓▓▓▓░░                                                
                                          ▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░▒▒▓▓▓▓▓▓▓▓▒▒▒▒                                                    
                                            ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒▓▓▓▓▒▒░░░░░░░░░░▒▒▓▓▓▓▓▓▓▓▓▓▓▓▒▒                                                    
                                              ▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▒░░▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒                                                  
                                            ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░░░▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░░░░░                                                
                                          ░░▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░▒▒▓▓▓▓▓▓▒▒▒▒░░░░░░░░░░░░                                                
                                          ░░░░░░▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒░░▒▒▓▓▒▒▒▒░░▒▒░░▒▒▒▒▒▒▓▓▓▓▓▓                                              
                                          ▒▒▒▒░░░░░░▒▒▓▓▓▓▓▓▓▓▓▓▒▒▓▓▓▓▓▓▓▓▓▓░░░░░░░░▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓░░                                            
                                        ▒▒▒▒▒▒▓▓▒▒░░░░░░▒▒▓▓░░▒▒▓▓▓▓▓▓▓▓▒▒░░░░▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓                                            
                                        ▓▓  ░░▓▓░░▒▒▒▒▒▒▒▒░░      ▓▓▓▓░░░░░░▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓                                            
                                        ░░  ▒▒  ▒▒  ▒▒▓▓▓▓          ░░▒▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▒░░                                          
                                                ░░  ▓▓░░░░              ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▒▒▒▒▒░░▒▒░░░░░░                                          
                                                    ░░▓▓░░                ▓▓▓▓▓▓▒▒░░░░░░░░░░░░░░░░░░░░░░                                          
                                                      ░░                    ▒▒▒▒░░░░▒▒░░░░▒▒▒▒▒▒▒▒▒▒▒▒░░                                          
                                                                              ░░▒▒░░░░▒▒▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓                                          
                                                                              ░░▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓                                          
                                                                              ░░▓▓▒▒▓▓░░▓▓  ▓▓░░▒▒░░▒▒░░░░                                        
                                                                                  ▒▒▓▓░░▓▓░░░░  ▓▓░░▓▓░░▓▓                                        
                                                                                  ▒▒  ░░░░░░░░  ▒▒  ▒▒  ▓▓

**Comment by tombui99** [comment]
Exciting news!

**Comment by AshAnil** [comment]
Exciting times for SSW 🦙🎉🎉

**Comment by JeanThirion** [comment]
Now I'm also very excited about the opportunity to sell Tina over in Europe. That's been on the back of my head for a while, and now we can start for real... @PothieuG - it's your time to shine :)

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33501311/aa755740-341a-4e82-b0ed-37fa2689c97a)

**Comment by MattParkerDev** [comment]
Tina come get some ham!

**Reply by bradystroud** [reply]
Haha I feel like no one knows the reason behind the Llama icon

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/38869720/398272ac-a4b0-4d45-b103-37156a53da97)

**Reply by bettybondoc** [reply]
Only a few people know. @landonmaxwell had to show me a clip a while back to understand why the llama is called Tina

**Comment by JackDevAU** [comment]
Sad I couldn't make it to ../New but it looked like a bunch of fun was had! 

It's been awesome working with the Tina Team so far - learning a lot and can't wait to ship some cool features!

**Comment by sethdaily** [comment]
https://github.com/SSWConsulting/SSW.Rules.Content/discussions/8557#discussioncomment-9463706

**Comment by arig2024** [comment]
Newcastle was rocking @ Tina launch on Wednesday evening.

**Comment by wicksipedia** [comment]
It has been really interesting taking consulting skills and using them in a product context

**Comment by stephanfakossw** [comment]
Interesting times ahead! Great to see our devs engaged and excited...
The Llama is the best thing ever 😀

**Comment by christoment** [comment]
Excited that the announcement went well and the party was a blast with a llama (is that alpaca or llama???)
To be honest I'm not yet convinced with the possibility of TinaCMS, but maybe time will prove me wrong 😊

**Comment by Calinator444** [comment]
It's exciting. I think markdown editors like these have the capacity to bridge the gap between marketing teams and software developers in controlling the content of the website.

**Comment by cvionnet** [comment]
This is an exciting time for SSW! 🌟🦙

**Comment by Aibono1225** [comment]
Looking forward to seeing the next step! 💘🦙

**Comment by jerryluossw** [comment]
This is an exciting time for SSW, looking forward to push the Chinafy work😀

**Comment by jernejk** [comment]
I haven't used it yet, so I don't have a lot of opinion on Tina CMS. 😊
Just curious on what will happen.

**Comment by babakamyljanovssw** [comment]
It is great news for SSW, excited for new features and try it out.

**Comment by hveraus** [comment]
I am looking forward to get this Llama Chinafyed 🦙 🦙 🐑

**Comment by Levijj22** [comment]
Very exciting time for SSW! 
Its an interesting change from selling expensive consulting to a free/low price product.
Loving the experience!

**Comment by bradystroud** [comment]
I love reviewing chicken parmis using TinaCMS

https://parmipicks.com

**Comment by PennyWalker** [comment]
I look forward to wearing llama everything, and some kind of Napolian dynamite t-shirt. :D

On a serious note, I'm really excited to see the amazing ideas from the Tina team. Scott and his team are going to make magic. 🪄 They already have an awesome product that we love, and now we can put our money where our mouth is to make it better!! A product with customers - here we come! Scary and exciting!

**Comment by miichaelsmedley** [comment]
Love the announcement but I wondered about the name... Tina? and the connection with Llamas. Quick Google tells me its from the Film Napoleon Dynamite - Maybe it should have some humour on the origins story on the website??

**Reply by bettybondoc** [reply]
Oh you're right, that might be cool so people could finally piece it together. Only a handful of people understood the reference haha.

**Comment by jimmidier** [comment]
Exciting moment!

**Comment by sylhuang** [comment]
Real Llama! Love it!

**Comment by danielmackay** [comment]
Initially, I wasn't a massive fan of TinaCMS, but @wicksipedia put on a very cool demo at the launch party which has now got me quite interested!

**Comment by tino-liu** [comment]
This is very exciting. I mainly used Sanity before... now, I'm dying to try out TinaCMS!

**Comment by amankumarrr** [comment]
I see a lot of potential in TinaCMS. So far, we've been using it from a developer's perspective, but a CMS should cater more to end users (non-IT people). This is where we need to focus our efforts.

**Comment by tiagov8** [comment]
Looking forward to seeing TinaCMS developing with SSW expertise (and many llama emojis)! I already noticed that the bug you could start editing on the main branch has been resolved ❤️ 🦙

**Comment by vladislav-kir** [comment]
Looking forward too see how Tina will grow with our experience!

**Comment by KostaMadorsky** [comment]
Amazing news! Can't wait to have some hands-on experience with tinaCMS.

**Comment by bahjatalaadelssw** [comment]
Let's make some videos!
